nomad/integration-tests-new.nomad

```diff
@@ -36,19 +36,19 @@ variable "pipeline_ingress_healthcheck_polling_interval_ms" {
   description = "The amount of time to wait between each healthcheck execution."
 }
 
-variable "pipeline_ingress_kafka_consumer_group_name" {
+variable "integration_tests_kafka_consumer_group_name" {
```

Comment:
I'd argue you don't need to necessarily add these integration_tests_ prefixes given the filename, but not that important. I'm fine with the env vars having it, though.